packages/firestore/test/unit/specs/write_spec.test.ts

```diff
@@ -996,4 +996,98 @@ describeSpec('Writes:', [], () => {
         .writeAcks('collection/b', 2000);
     }
   );
+
+  specTest('Secondary tabs handle user change', ['multi-client'], () => {
+    const query = Query.atPath(path('collection'));
+    const docALocal = doc(
+      'collection/a',
+      0,
+      { v: 1 },
+      { hasLocalMutations: true }
+    );
+    const docBLocal = doc(
+      'collection/b',
+      0,
+      { v: 1 },
+      { hasLocalMutations: true }
+    );
+    const docCLocal = doc(
+      'collection/c',
+      0,
+      { v: 1 },
+      { hasLocalMutations: true }
+    );
+
+    // Firebase Auth attempts to rapidly synchronize user changes across tab. We
+    // emulate this behavior in this spec tests by calling `changeUser` manually
+    // for all clients.
+    return (
```

Comment:
Is this paren necessary?  My brain explodes trying to find the match.

Reply:
I believe `prettier` adds them. Let me remove it and see if it gets re-added :)